action.yml

```diff
@@ -39,17 +35,17 @@ outputs:
 runs:
   using: 'composite'
   steps:
-    - name: 'Install Python'
-      uses: actions/setup-python@v5
-      id: setup-python
-      with:
-        python-version: ${{ inputs.python_version }}
```

Comment:
Is there a way of marking this as deprecated or something? will it crash if you have an action using this?

Reply:
Done, added deprecation message in the README and in the action. You can check the output here: https://github.com/conan-io/setup-conan/actions/runs/15345935485/job/43182035157?pr=5#step:3:8